src/plugin/physics/src/utils/CollisionUtils3D.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "Math.hpp"
+
+#include "BoxCollider3D.hpp"
+
+namespace ES::Plugin::Physics::Utils {
+/**
+ * @brief Check if two 3D boxes are colliding
+ *
+ * @param   posA   The position of the first box
+ * @param   boxA       The size of the first box
+ * @param   posB   The position of the second box
+ * @param   boxB       The size of the second box
+ * TODO: change fn name
```

Comment:
still the case?

Reply:
no it is fine now
I removed the comment